helpers/numbers.ts

```diff
@@ -18,3 +18,9 @@ export function price(x: string): BigNumber {
 export function usdc(x: string): BigNumber {
   return fp(x, 6);
 }
+
+export function tokenWithNDecimalsPrecisionScalar(n: number): BigNumber {
+  if (n > 18 || n < 0) throw new Error(`Invalid n given: ${n}`);
```

Comment:
Can we add brackets here so that the styling remains consistent across the board?

Reply:
done